pkg/server/gateway.go

```diff
@@ -52,6 +53,12 @@ const (
 	RetryInterval = 10 * time.Millisecond
 )
 
+var kacp = keepalive.ClientParameters{
```

Comment:
nit: I generally prefer to have variable names that are long and obvious instead of short and obscure, so `keepAliveClientParams` would be preferable over `kacp`.

Reply:
Have updated the vars